server/cla.go

```diff
@@ -121,3 +130,15 @@ func checkCLAComment(comments []*github.IssueComment, username string) (int64, b
 	}
 	return 0, false
 }
+
+func isUserExcluded(exclusions []string, username string) bool {
+	if len(exclusions) == 0 {
+		return false
+	}
```

Comment:
Is this really needed? If `len(exclusions) == 0` the `for` loop won't run anyway.